### PR TITLE
feat(lambda): adding support for non-error 1st callback arg

### PR DESCRIPTION
**Pull request by galbash**

### CODE REVIEW

**Review by ranrib — DISMISSED** [review]

File: src/wrappers/lambda.js [hunk 1]
```diff
@@ -97,7 +97,21 @@ function baseLambdaWrapper(
...
             if (error) {
                 // not catching false here, but that seems OK
-                eventInterface.setException(runner, error, false);
+                let reportedError = error;
+                if (!error.name) {
+                    let errorMessage;
+                    try {
+                        errorMessage = typeof error === 'string' ? error : JSON.stringify(error);
+                    } catch (stringifyErr) {
+                        errorMessage = 'Unserializable Error';
```
Comment (adavidai) [inline src/wrappers/lambda.js]: Avoid overriding the user error with an error in the instrumentation itself

File: test/unit_tests/wrappers/test_lambda.js [hunk 2]
```diff
@@ -258,6 +258,25 @@ describe('lambdaWrapper tests', () => {
         }, 1);
     });
 
+    it('lambdaWrapper: wrapped callback string as error', (done) => {
```
Comment (adavidai) [inline test/unit_tests/wrappers/test_lambda.js:261]: Consider enhancing the tests to include other object types for error

**Review by adavidai — COMMENTED** [review]
Looks good 👍 
See some comments below

**Review by yotamelkaslasy — APPROVED** [review]

**Pull request merged**